app/helpers/file_icons_helper.rb

```diff
@@ -4,34 +4,20 @@ def wopi_file?(asset)
     %w(csv ods xls xlsb xlsm xlsx odp pot potm potx pps ppsm ppsx ppt pptm pptx doc docm docx dot dotm dotx odt rtf).include?(file_ext)
   end
 
-  # For showing next to file
   def file_fa_icon_class(asset)
     file_ext = asset.file_file_name.split('.').last
-    if Constants::FILE_TEXT_FORMATS.include?(file_ext)
-      fa_class = 'fa-file-word'
+
+    if Extends::FILE_FA_ICON_MAPPINGS[file_ext] # Check for custom mappings or possible overrides
+      return Extends::FILE_FA_ICON_MAPPINGS[file_ext]
+    elsif Constants::FILE_TEXT_FORMATS.include?(file_ext)
+      return 'fa-file-word'
     elsif Constants::FILE_TABLE_FORMATS.include?(file_ext)
-      fa_class = 'fa-file-excel'
+      return 'fa-file-excel'
     elsif Constants::FILE_PRESENTATION_FORMATS.include?(file_ext)
-      fa_class = 'fa-file-powerpoint'
+      return 'fa-file-powerpoint'
     elsif %w(pdf).include?(file_ext)
-      fa_class = 'fa-file-pdf'
+      return 'fa-file-pdf'
     elsif %w(txt csv tab tex).include?(file_ext)
-      fa_class = 'fa-file-alt'
-    end
-
-    # Now check for custom mappings or possible overrides
-    if Extends::FILE_ICON_MAPPINGS[file_ext]
```

Comment:
@okriuchykhin 
Was this mistake or I don't get the context here. You are checking `FILE_ICON_MAPPINGS ` but then assigning a value from `FILE_FA_ICON_MAPPINGS ` hash. 
If this is correct code, mine is wrong.

Reply:
Looks like a type, should be FILE_FA_ICON_MAPPINGS